readthedocs/projects/tasks/builds.py

```diff
@@ -744,7 +744,11 @@ def after_return(self, status, retval, task_id, args, kwargs, einfo):
         self.update_build(build_state)
         self.save_build_data()
 
-        build_complete.send(sender=Build, build=self.data.build)
+        # Be defensive with the signal, so if a listener fails we still clean up
+        try:
+            build_complete.send(sender=Build, build=self.data.build)
+        except Exception:
+            log.exception("Error during build_complete", exc_info=True)
```

Comment:
the exception should be included automatically

```suggestion
            log.exception("Error during build_complete")
```